keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -196,46 +196,48 @@ class SqlVerificationRepository(
      * This function guarantees that the number of output elements match the number of input elements.
      * So we use left joins on the givenVersions table
      */
-    return jooq.select(
+    return contextTable.table?.let { ctxTable ->
```

Comment:
nit: I think this might be a bit easier to reason about if you added an early return here like so:
```kotlin
return if contextTable.table == null {
  emptyList()
} else {
  // select code
}
```